webviz_subsurface/plugins/_seismic_misfit.py

```diff
@@ -0,0 +1,3023 @@
+# pylint: disable=too-many-lines, too-many-arguments, too-many-locals
+from typing import List, Dict, Any, Optional, Union, Tuple
+from pathlib import Path
+import logging
+import re
+import glob
+import pandas as pd
```

Comment:
```suggestion
import re
import glob

import pandas as pd
```
Importes are grouped according to:
1. standard library
2. third-party packages
3. internal modules in same package

Reply:
Ok, thanks. Added some spacing